pkg/commands/os.go

```diff
@@ -175,7 +175,8 @@ func (c *OSCommand) Unquote(message string) string {
 	return message
 }
 
-func (C *OSCommand) AppendLineToFile(filename, line string) error {
+// AppendLineToFile adds a new line at the beginning of a given string
```

Comment:
It actually adds the given string as a new line to the given file :)

Reply:
yep that's stupid